content/applications/general/iot/config/troubleshooting.rst

```diff
@@ -343,6 +343,66 @@ Save`.
              :align: center
              :alt: Receipt format using naming convention: EpsonTMU220B__IMC_LDV_LDH_SCALE35__.
 
+DYMO LabelWriter 450 DUO print delay
+------------------------------------
+
+The DYMO LabelWriter 450 DUO has a known delay in printing with the :abbr:`IoT (Internet of Things)`
+box while using the label function. This is due to the printer being set up as a *Local RAW
+Printer*. To resolve this issue, open the OpenPrinting CUPS administration page from the :abbr:`IoT
+(Internet of Things)` box homepage and click :guilabel:`Add a Printer`.
+
+.. important::
+   The DYMO LabelWriter 450 DUO printer must already be connected and installed on the :abbr:`IoT
+   (Internet of Things)` box.
+
+.. image:: troubleshooting/add-printer-dymo.png
+   :align: center
+   :alt: Add a printer button highlighted on the Printer CUPS management page.
```

Comment:
same remark as the previous one, the printer is automatically seen with cups (as long as it is connected to the IoT box) & the "raw" driver is automatically given to it. You don't need to add it manually